ingress-nginx-controller-1.11.yaml

```diff
@@ -3,7 +3,7 @@ package:
   name: ingress-nginx-controller-1.11
   version: 1.11.3
   # There are manual changes to review between each package update. See 'vars:' section.
-  epoch: 1
+  epoch: 2
```

Comment:
Discovered this was FTBFS. Tests were failing:

```bash
    test:
      pipeline:
        - runs: |
            mkdir -p /modules_mount/etc/nginx/modules/otel
            /usr/bin/init_module
            stat /modules_mount/etc/nginx/modules/otel/otel_ngx_module.so
```

Was failing to find otel_ngx_module.so. That is only copied into place by the non-compat package (opentelemetry-plugin-nginx):
- https://github.com/wolfi-dev/os/blob/main/opentelemetry-plugin-nginx.yaml#L72

Added as a runtime dependency